protocol/avdl/update_common.avdl

```diff
@@ -24,6 +24,7 @@ protocol updateCommon {
     string version; // Sematic version, e.g. 1.2.3-400
     string name;
     string description;
+    union { null, string } instructions; // String representing distro-specific upgrade instructions
```

Comment:
I think using default empty string for empty (instead of union null) is pretty common in the protocol?